flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/job/RescalingStatusMessageParameters.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.handler.job;
+
+import org.apache.flink.runtime.rest.messages.JobMessageParameters;
+import org.apache.flink.runtime.rest.messages.MessagePathParameter;
+import org.apache.flink.runtime.rest.messages.TriggerIdPathParameter;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * Message headers for the {@link RescalingHandlers.RescalingStatusHandler}.
+ */
+public class RescalingStatusMessageParameters extends JobMessageParameters {
+
+	public final TriggerIdPathParameter triggerIdPathParameter = new TriggerIdPathParameter();
+
+	@Override
+	public Collection<MessagePathParameter<?>> getPathParameters() {
+		return Arrays.asList(jobPathParameter, triggerIdPathParameter);
```

Comment:
It's not very important but the `List` returned by `Arrays.asList` is mutable:

```
	public static void main(String[] args) {
		final List<String> strings = Arrays.asList("0", "1", "2");
		strings.set(2, "3");

		System.out.println(strings);
		System.out.flush();

		final List<String> strings2 = Collections.unmodifiableList(strings);
		strings2.set(2, "2");
	}
```

output
```
[0, 1, 3]
Exception in thread "main" java.lang.UnsupportedOperationException
	at java.util.Collections$UnmodifiableList.set(Collections.java:1311)
	at org.apache.flink.runtime.dispatcher.Dispatcher.main(Dispatcher.java:347)
```

Reply:
True, there will be other places to fix as well. I'll leave it for a follow up task.